docs/tools/vdb_table/data/couchbase.json

```diff
@@ -0,0 +1,181 @@
+{
+  "name": "Couchbase",
+  "links": {
+    "docs": "https://docs.couchbase.com/cloud/vector-search/vector-search.html",
+    "github": "https://github.com/couchbase",
+    "website": "https://www.couchbase.com/products/vector-search/",
+    "vendor_discussion": "https://github.com/superlinked/VectorHub/discussions/xx",
+    "poc_github": "https://github.com/1tylermitchell",
+    "slug": "couchbase"
+  },
+  "oss": {
+    "support": "partial",
+    "source_url": "https://www.couchbase.com/blog/couchbase-adopts-bsl-license/",
+    "comment": ""
+  },
+  "license": {
+    "value": "Proprietary",
+    "source_url": "https://www.couchbase.com/blog/couchbase-adopts-bsl-license/",
+    "comment": "Vector search currently only part of Enterprise and DBaaS editions"
+  },
+  "dev_languages": {
+    "value": [
+      "Python", "c++", "erlang", "go", "java"
```

Comment:
please limit this list to the 1 or 2 main languages that the DB is written in. Like Daniel mentioned, they need to be in lowercase as specified in the enum here: https://github.com/superlinked/VectorHub/blob/49931762584d2a4f91cbc063d348dc390563a310/docs/tools/vdb_table/vendor.schema.json#L86-L97